### PR TITLE
Various

### PR DESCRIPTION
**Pull request by aucuparia**

### CODE REVIEW

**Review by lwesterhof — APPROVED** [review]
LGTM

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

File: controllers/Datarequest.php [hunk 1]
```diff
@@ -15,85 +15,120 @@ public function __construct() {
...
-        $isRequestOwner = $this->api->call('datarequest_is_owner', ['request_id' => $requestId])->data;
-        $isReviewer     = $this->api->call('datarequest_is_reviewer', ['request_id' => $requestId])->data;
+        $isProjectManager    = $this->api->call('datarequest_is_role', ["role" => "PM"])->data;
+        $isExecutiveDirector = $this->api->call('datarequest_is_role', ["role" => "ED"])->data;
+        $isDatamanager       = $this->api->call('datarequest_is_role', ["role" => "DM"])->data;
+        $isDMCMember         = $this->api->call('datarequest_is_role', ["role" => "DMC"])->data;
+        $isRequestOwner      = $this->api->call('datarequest_is_role',
+                                                ["role" => "OWN", "request_id" => $requestId])->data;
+        $isReviewer          = $this->api->call('datarequest_is_role',
+                                                ["role" => "REV", "request_id" => $requestId])->data;
```
Comment (lwesterhof) [inline controllers/Datarequest.php:73]: Can we do this more efficient? We now have to do 6 API calls to get the role of the user. Can we get this information when loading the form or combine these and return a single JSON structure with the user group memberships and statuses?

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: controllers/Datarequest.php (hunk 1 above)
Comment (aucuparia) [reply controllers/Datarequest.php:73]: True. Will fix.

Comment (aucuparia) [reply controllers/Datarequest.php:73]: Fixed.

**Review by lwesterhof — APPROVED** [review]

**Pull request merged**